templates/image.jsx

```diff
@@ -49,9 +49,7 @@ export default function Image(props) {
 
       {props.attribution &&
       <span className={prefixClasses(attributionClassNamePrefixes, ['__attribution'])}>
-        <span className={prefixClasses(attributionClassNamePrefixes, ['__attribution-inner'])}>
-          {html(props.attribution)}
-        </span>
+        <span className={prefixClasses(attributionClassNamePrefixes, ['__attribution-inner'])} dangerouslySetInnerHTML={{ __html: compile(props.attribution, props) }} />
```

Comment:
Not really a requirement, more of a preference. Up to you if you'd like to include.
```suggestion
        <span 
          className={prefixClasses(attributionClassNamePrefixes, ['__attribution-inner'])} 
          dangerouslySetInnerHTML={{ __html: compile(props.attribution, props) }} 
        />
```